components/delivery-security-instructions.spec.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DeliverySecurityInstructions } from './delivery-security-instructions.jsx';
+
+describe('DeliverySecurityInstructions', () => {
+	describe('renders component', () => {
+		let wrapper;
+
+		beforeEach(() => {
+			wrapper = shallow(
+				<DeliverySecurityInstructions />
+			);
+		});
+
+		test('contains textAreaProps id set to default: deliverySecurityInstructionsField ', () => {
+			expect(wrapper.prop('id')).toBe('deliverySecurityInstructionsField');
+		});
+
+		test('contains id set to foo', () => {
+			wrapper = shallow(
+				<DeliverySecurityInstructions fieldId='foo'/>
+			);
+			expect(wrapper.prop('id')).toBe('foo');
+		});
+
+		test('contains htmlFor set to default: deliverySecurityInstructions', () => {
+			expect(wrapper.prop('htmlFor')).toBe('deliverySecurityInstructions');
+		});
+
+		test('contains htmlFor set to foo', () => {
+			wrapper = shallow(
+				<DeliverySecurityInstructions inputId='foo'/>
+			);
+			expect(wrapper.prop('htmlFor')).toBe('foo');
+		});
+
+		test('containing o-forms-input--invalid when hasError is set to true', () => {
+			wrapper = shallow(
+				<DeliverySecurityInstructions hasError={true}/>
+			);
+			const secondChildProps = wrapper.props().children[1].props;
+
+			expect(secondChildProps.className).toContain('o-forms-input--invalid');
+		});
+
+		test('does not contain o-forms-input--invalid when hasError is false', () => {
+			const secondChildProps = wrapper.props().children[1].props;
+			expect(secondChildProps.className).not.toContain('o-forms-input--invalid');
+		});
+
+		describe('sets default props for textarea', () => {
+			const wrapper = shallow(<DeliverySecurityInstructions />);
+			const textAreaProps = wrapper.find('textarea').props();
+
+			it('sets id to deliverySecurityInstructions', () => {
+				expect(textAreaProps.id).toBe('deliverySecurityInstructions');
+			});
+
+			it('sets name to deliverySecurityInstructions', () => {
+				expect(textAreaProps.name).toBe('deliverySecurityInstructions');
+			});
+
+			it('does not set maxLength when no value is given', () => {
+				expect(textAreaProps.maxLength).toBeUndefined;
+			});
+
+			it('does not set rows when no value is given', () => {
+				expect(textAreaProps.rows).toBeUndefined;
+			});
+
+			it('sets data-trackable to field-deliverySecurityInstructions', () => {
+				expect(textAreaProps['data-trackable']).toBe('field-deliverySecurityInstructions');
+			});
+
+			it('sets placeholder to: Please enter any secure information here, e.g. security gate access codes', () => {
```

Comment:
Not a big deal but I would advise against putting the actual value in the test description (unless your test results depend specifically on variations of said value).
I makes code harder to maintain/change and more error prone (you update the value used in the test but forget or make a mistake when updating the description).

```suggestion
			it('sets the `placeholder` property', () => {
```